src/utils/requests/workOrders.ts

```diff
@@ -1,7 +1,7 @@
 import { frontEndApiRequest } from '@/utils/frontEndApiClient/requests'
 import { WorkOrder } from '@/models/workOrder'
 import { ApiResponseType } from '../../types/requests/types'
-import { NoteData } from '../../types/edit-workorder/types'
+import { NoteData } from '../../types/generic/types'
```

Comment:
Not sure about a generic path. I would keep the type as close to the relevent logic, unless it truely is generic. How about something like `/src/utils/requests/types.ts`.